Scripts/uetorch.lua

```diff
@@ -508,6 +510,7 @@ uetorch.SetActorVisible = utlib.SetActorVisible
 uetorch.SetActorVelocity = utlib.SetActorVelocity
 uetorch.SetActorAngularVelocity = utlib.SetActorAngularVelocity
 uetorch.SetActorScale3D = utlib.SetActorScale3D
+uetorch.DestroyActor = utlib.DestroyActor
```

Comment:
I guess this one is indeed pretty particular to the project, and it wouldn't be hard to include if it's necessary later on.